docs/source/notes/production_ecosystem.rst

```diff
@@ -0,0 +1,131 @@
+Integration in production ecosystem
+===================================
+
```

Comment:
Add a table of contents here

Reply:
There's already a table of contents rendered on the right, but sure